lib/handlerTemplate.js

```diff
@@ -192,7 +192,11 @@ function prepareRequestChain(conf) {
         if (requestConf.request) {
             template = new Template(requestConf.request);
             requestSpec.request = function(restbase, context) {
-                var req = template.eval(context);
+                var req = template.expand(context);
+                if (!requestConf.request.method) {
+                    // Do we really want this?
+                    req.method = context.request.method || 'get';
```

Comment:
I like the current behaviour actually. If there, use the provided method. Otherwise, use the original request's method (effectively reusing the route's method from the spec). Finally, use `GET`. It makes the most sense to me.

I can't see an added benefit in making the method explicit in the spec. Every time.